### ISSUE TITLE
Boss Selection

### ISSUE THREAD

**Issue by Bergman777** [Azurblau/Zombies.Zone#142]
Whenever i join zombie team i never get selected as boss. you basically have to main zombie from the start and then you most likely get selected as boss for every single round after that which is quite unfair. i would like it to randomly select someone as a boss instead. or atleast make it so that you can only be boss once per game unless you are alone or the other zombies also already have been boss once.

**Comment by Azurblau** [comment]
Please keep in mind that the tracking of your progress for getting Boss Zombie is now wave-based, not round-based. So you have to be the best zombie of a wave to become boss. Since this ticket is about a different feature, limiting boss to once per player, I'll keep it open and wait for more opinions.

**Comment by Azurblau** [comment]
See also #34

**Comment by Loickun** [comment]
Yes indeed. Bergman777 what you need to know is that the damage you deal as zombie towords Humans counts 100% and the damage u deal to barricades only 20% that means if u hit a pile of humans as zombie you have more chances to become zombie boss. 

Forcing every zombo player to play agressive as possible.

**Comment by Loickun** [comment]
About that 2nd idea of yours. Making it so that the same player dont gets Boss again could be something I like. So I'd give that a +1  (Dont get boss multiple times for the same player unless he is alone)

Wich would also cause zombie team to stock up on boss zombies if the boss from wave 2 and 3 would still be alive.
